src/main/java/org/apache/xml/security/encryption/XMLCipherUtil.java

```diff
@@ -302,4 +338,30 @@ public static ConcatKDFParams constructConcatKeyDerivationParameter(int keyBitLe
         kdp.setSuppPrivInfo(suppPrivInfo);
         return kdp;
     }
+
```

Comment:
I guess we should add javadoc here to align with the concat static method?

Reply:
Done